api/api/react.py

```diff
@@ -1,7 +1,9 @@
 from django.conf import settings
 import os
+from mapping.models import Status
 def react(request):
     return {
         "a": os.environ.get("COCONNECT_DB_AUTH_TOKEN"),
-        "u": os.environ.get("CCOM_APP_URL")     
+        "u": os.environ.get("CCOM_APP_URL")   
+        "status":[{'id': year[0], 'label': year[1]} for year in Status.choices]
```

Comment:
Please remove the `year` here which is not relevant. Either 
`[{'id': choice[0], 'label': choice[1]} for choice in Status.choices]` or (even better) `[{'id': id, 'label': label} for id, label in Status.choices]`